src/Sarif.FunctionalTests/DirectProducerTestData/CredentialScanner/scan-suppressed.sarif.json

```diff
@@ -0,0 +1,236 @@
+            ]
+          },
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "file:///D:/src/CredentialScanner/CredentialScannerTests/files/powershell.ps1",
+                "region": {
+                  "startLine": 9
+                }
+              }
+            }
+          ],
+          "toolFingerprintContribution": "NgSTwJtXGCiqdoY1o5FNT/9IiPhjX1MVXIMGkRIg9AI=",
+          "suppressionStates": ["suppressedInSource"]
+        },
+        {
+          "ruleId": "SCAN01",
+          "level": "note",
+          "formattedRuleMessage": {
+            "formatId": "Default",
+            "arguments": [
+              "storagecreds.config",
+              "Config-Files",
+              "Found .config file matching known credential pattern.",
+              "Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan."
+            ]
+          },
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "file:///D:/src/CredentialScanner/CredentialScannerTests/files/storagecreds.config",
+                "region": {
+                  "startLine": 18
+                }
+              }
+            }
+          ],
+          "toolFingerprintContribution": "uf/5ni/gNQZM2xLIff7J0/yJxmvpvsL2fQ7VW9XvxLM=",
+          "suppressionStates": ["suppressedInSource"]
+        },
+        {
+          "ruleId": "SCAN01",
+          "level": "note",
+          "formattedRuleMessage": {
+            "formatId": "Default",
+            "arguments": [
+              "storagecreds.config",
+              "Config-Files",
+              "Found .config file matching known credential pattern.",
+              "Validate file contains secrets, remove, roll credential, and use approved store. For additional information on secret remediation see https://aka.ms/credscan."
+            ]
+          },
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "file:///D:/src/CredentialScanner/CredentialScannerTests/files/storagecreds.config",
+                "region": {
+                  "startLine": 20
+                }
+              }
+            }
+          ],
+          "toolFingerprintContribution": "DDF46ymA/XcWHjXcD9RwpnR4vuNFZs10WoHYQoRsDLs=",
+          "suppressionStates": ["suppressedInSource"]
+        }
+      ],
+      "rules": {
+        "SCAN01": {
+          "id": "SCAN01",
+          "name": "SecretDetectedInFile",
+          "shortDescription": "A potential secret was detected within a file.",
+          "fullDescription": "A potential secret was detected within a file.",
+          "messageFormats": {
+            "Default": "A potential secret was detected in '{0}':({1}) {2} {3}"
+          },
+          "defaultLevel": "error",
+          "helpUri": "https://aka.ms/credscan"
```

Comment:
Is CredScan a Microsoft-internal tool? Even if not, isn't this a Microsoft-internal URL? Do we want this in a public repo?